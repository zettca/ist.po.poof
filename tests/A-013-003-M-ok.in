1
6
1
root
ROOT
0
0
