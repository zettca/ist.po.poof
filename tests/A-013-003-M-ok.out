Menu Principal
1 - Novo
2 - Abrir
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Identificador do utilizador: Nome do utilizador: Criar utilizador: Operação inválida: O utilizador 'root' já existe.
Menu Utilizador
1 - Criar utilizador
2 - Listar utilizadores
0 - Sair
Escolha uma opção: Menu Principal
1 - Novo
2 - Abrir
3 - Guardar
4 - Login
5 - Menu Shell
6 - Menu Utilizador
0 - Sair
Escolha uma opção: 